composer.json

```diff
@@ -18,8 +18,7 @@
     "require-dev": {
         "composer/composer": "1.0.*@dev",
         "jakub-onderka/php-parallel-lint": "~0.8",
-        "phpspec/prophecy-phpunit": "~1.0",
-        "phpunit/phpunit": "~4.0",
+        "phpunit/phpunit": "~4.8|~5.0",
```

Comment:
Using a travis matrix it is possible to use phpunt 4 with php 5.3 / 5.4 / 5.5 and phpunit 5 with php 5.6 / 7 

I saw some example recently... but don't remember in which project...:(